TGM/src/main/java/network/warzone/tgm/command/PunishCommands.java

```diff
@@ -229,6 +229,30 @@ public static void staffchat(CommandContext cmd, CommandSender sender) {
         Bukkit.getConsoleSender().sendMessage(result);
     }
 
+    @Command(aliases = {"chat"}, desc = "Control chat settings", min = 1, usage = "(mute|clear)")
+    @CommandPermissions({"tgm.chat.control"})
+    public static void chat(CommandContext cmd, CommandSender sender) {
+        String action = cmd.getString(0);
+        if (action.equalsIgnoreCase("mute")) {
+            if (TGM.get().getConfig().getBoolean("chat.enabled")) {
+                TGM.get().getConfig().set("chat.enabled", false);
+                TGM.get().saveConfig();
+                Bukkit.broadcastMessage(ChatColor.DARK_AQUA + sender.getName() + " muted the chat.");
+            } else {
+                TGM.get().getConfig().set("chat.enabled", true);
+                TGM.get().saveConfig();
+                Bukkit.broadcastMessage(ChatColor.DARK_AQUA + sender.getName() + " unmuted the chat.");
+            }
+        } else if (action.equalsIgnoreCase("clear")) {
+            for (Player player : Bukkit.getOnlinePlayers()) {
+                for(int i=0; i <= 100; i++){
```

Comment:
Instead of sending a ton of messages and spamming clients with packets, we should use StringUtils' repeat string method and use `\n`. This could also be done using StringBuilder but initialize it before getting the online players.